main.go

```diff
@@ -151,6 +152,9 @@ func (br *WABridge) Start() {
 		br.Provisioning.Init()
 	}
 	go br.CheckWhatsAppUpdate()
+	if br.Config.Homeserver.Software == bridgeconfig.SoftwareHungry {
```

Comment:
Can we move this gate further down? Maybe check right in `puppet.UpdateContactInfo` so that we don't have to remember to gate every call